server/src/main/resources/resourceBundles/messages_en.properties

```diff
@@ -85,4 +85,6 @@ variables.integer=integer
 variables.blankWhenZero=BLANK WHEN ZERO
 variables.signClause=SIGN CLAUSE
 variables.mapName=Map name
+IdmsCopybookVisitor.errorCircularDependency=%s: Copybook has circular dependency
```

Comment:
@zeibura can you help with the text?

Reply:
Maybe change it to "Copybook has circular dependencies" but its fine really.